### PR TITLE
Fixed English & French translations, Proposal for Mail AntiSpam

### PR DESCRIPTION
**Pull request by winkelsdorf**
English: Withdrawal, not Revocation (also for the forms)
French: Rétractation, c’est ne pas révocation (aussi pour les
formulaires)

French sample withdrawal form:

Modèle de formulaire de rétractation
http://www.legifrance.gouv.fr/affichCode.do?cidTexte=LEGITEXT000006069565&idSectionTA=LEGISCTA000029472300&dateTexte

### CODE REVIEW

**Comment by winkelsdorf** [comment]
By the way: 

1) Is there a best practice of updating the templates within an installed mageSetup? Speaking of having the old templates already in place on my test site and I want to update them without editing every single template..
2) Travis CI broken somehow (it's using magento 1.7.x).

**Comment by winkelsdorf** [comment]
Anyone?

**Comment by Schrank** [comment]
Thank you for your effort! Too much for 0:30 to go through, will do it the next days.

**Comment by winkelsdorf** [comment]
Nevermind, this has no hurry. Translations are probably fine to accept as I did them carefully, the bug fix for the AntiSpam in Mail Templates is something that needs testing - despite it "works on my machine" ;) For a quick review see: https://github.com/winkelsdorf/firegento-magesetup/commit/48d95d0775218bb882ca151b4bb67564a11d2ad9

**Comment by riker09** [comment]
Tested in a live environment, seems to work pretty well.

File: src/app/locale/fr_FR/FireGento_MageSetup.csv [hunk 1]
```diff
@@ -24,7 +24,7 @@
 "Update System Config Settings","Mettre à jour les réglages de la configuration système"
 "i.e. newsletter confirmation, account confirmation, ..", "expl. confirmation de newsletter, confirmation de création de compte, ..."
 "Create CMS Pages and Blocks","Créer les pages CMS et blocs"
-"i.e. shipping, conditions, revocation","expl. livraisons, conditions, révocation"
+"i.e. shipping, conditions, revocation","expl. livraisons, conditions, rétractation"
```
Comment (jacquesbh) [inline src/app/locale/fr_FR/FireGento_MageSetup.csv]: `ex.` instead of `expl.`

Comment (jacquesbh) [inline src/app/locale/fr_FR/FireGento_MageSetup.csv]: `shipping` => `livraison`, if you want the same plurality.

Comment (winkelsdorf) [inline src/app/locale/fr_FR/FireGento_MageSetup.csv]: @jacquesbh Thanks for the correction, of course you are right. I want to note that these are words I didn't touch at all (been wrong before). Currently I am busy with another project, can somebody please accept this PR, and commit the proposed changes for jacquesbh?

**Comment by winkelsdorf** [comment]
@riker09 As said in the other thread: Thank you!

I leave the PR open for your convenience as it still fixes a Bug I reported (anti-spam in mails: https://github.com/winkelsdorf/firegento-magesetup/commit/48d95d0775218bb882ca151b4bb67564a11d2ad9). But to be honest, after 6 month I am actually not working on this fix anymore. It is used in a productive environment without issues since then. 

Probably a good idea to incorporate the changes @jacquesbh proposed, too.

File: src/app/locale/fr_FR/FireGento_MageSetup.csv (hunk 1 above)
Comment (jacquesbh) [inline src/app/locale/fr_FR/FireGento_MageSetup.csv]: I know.
But this PR is a good opportunity to fix the "wrong before" :+1:

**Comment by winkelsdorf** [comment]
All changes from @jacquesbh incorporated. Ready to merge (if still of interest 8 months after).

**Comment by Schrank** [comment]
Woho! Made it.

Thank you very much!

**Comment by winkelsdorf** [comment]
Haha, good things are worth waiting for ^^ 

Glad to hear that, you're welcome!

**Pull request merged**